crates/ide/src/doc_links.rs

```diff
@@ -447,16 +489,14 @@ fn get_doc_base_url(db: &RootDatabase, def: Definition) -> Option<Url> {
             | LangCrateOrigin::ProcMacro
             | LangCrateOrigin::Std
             | LangCrateOrigin::Test),
-        ) => {
-            format!("https://doc.rust-lang.org/{channel}/{origin}")
-        }
-        CrateOrigin::Lang(_) => return None,
+        ) => (Some(format!("https://doc.rust-lang.org/{channel}/{origin}")), None),
```

Comment:
I believe the local docs come pre-installed right? We should be able to point this to `{sysroot}/share/doc/rust/{origin}/index.html`, we'd need to thread the sysroot base path down here as well then.